Adyen/UI/Form/Items/Address/FormFullAddressItem.swift

```diff
@@ -166,9 +177,32 @@ public final class FormAddressItem: FormValueItem<PostalAddress, AddressStyle>,
         case .stateOrProvince:
             observers[.stateOrProvince] = bind(item.publisher, to: self, at: \.value.stateOrProvince)
             publisherObservers[.stateOrProvince] = observe(publisher) { item.value = $0.stateOrProvince ?? "" }
+        case .country:
+            break
+        }
+    }
+    
+    private func updateItems(identifier: String?) {
+        let items = items + [headerItem, countrySelectItem]
+        for item in items {
+            if let item = item as? FormSplitItem {
+                update(item: item.leftItem, identifier: identifier)
+                update(item: item.rightItem, identifier: identifier)
+                continue
+            }
```

Comment:
this can be generic using `FormItem.subitems`:

```Swift
for item in items {
             for subitem in item.subitems {
                  update(item: subitem, identifier: identifier)
             }
             update(item: item, identifier: identifier)
}
```

Reply:
Improved